src/Uno.Foundation/Extensions/LinkerAttributes.cs

```diff
@@ -3,7 +3,7 @@
 
 #nullable enable
 
-#if NETSTANDARD2_0
+#if !NET6_0_OR_GREATER || NETSTANDARD2_0
```

Comment:
Is `NETSTANDARD2_0` necessary?

If we are on netstandard2.0, I think `!NET6_0_OR_GREATER` will already be true.